lib/ansible/modules/cloud/amazon/s3_sync.py

```diff
@@ -204,14 +208,14 @@
 # import module snippets
 import ansible.module_utils.ec2
```

Comment:
agree. I do have a preference for not importing functions into mainspace but I'm fine with this- at least it isn't `import * from x`.